### ISSUE TITLE
Setting up fake worker failed (Cypress)

### ISSUE THREAD

**Issue by severinkl** [stephanrauh/ngx-extended-pdf-viewer#982]
**Check this first**
The project is a wrapper around Mozilla's PDF viewer, so maybe your bug is caused by the base library. Please check this first by answering these three questions:
- Does the bug also happen on https://mozilla.github.io/pdf.js/web/viewer.html? (If so, please open the issue at Mozilla's bugtracker). **NO**
- Does the bug also happen on https://pdfviewer.net/simple? **YES**
- Can you write a reproducer based on this template project? https://github.com/stephanrauh/ngx-extended-pdf-viewer-issues/tree/main/template <-- I can create an example cypress project if necessary.

**Describe the bug**
When accessing the current version of the showcase with the current version of Cypress the pdf-viewer does not work.

**Version info**
- 10.1.0

**To Reproduce**
* Start a basic Cypress project (latest version 8.6.0)
* Select Chrome 94 as Browser (probably reproducible with other browsers as well)
* Visit https://pdfviewer.net/extended-pdf-viewer/simple
* The error below appears

**Screenshots**
![image](https://user-images.githubusercontent.com/22167134/137768215-5e305de2-bad6-48a3-a22f-2df12ae35b7a.png)

```PDF.js v2.10.558 (build: 0b46e8a77)
Message: Setting up fake worker failed: "Cannot load script at: https://pdfviewer.net/assets/pdf.worker.js".
```

**Additional context**
It seems like `pdf.worker.js` is missing in the assets and was removed at some point.

```bash
➜  ngx-extended-pdf-viewer git:(main) ✗ cd projects/ngx-extended-pdf-viewer/assets        
➜  assets git:(main) ✗ git log --diff-filter=D --summary | grep delete | grep pdf.worker.js
 delete mode 100644 projects/ngx-extended-pdf-viewer/src/assets-2.6/pdf.worker.js
 delete mode 100644 projects/ngx-extended-pdf-viewer/src/assets/pdf.worker.js.map
```

**Comment by severinkl** [comment]
Ok, thanks for the fast update! :pray: I hope I find time for it soon, but it's on my list.

Maybe I will also check out an earlier version of the `ngx-extended-pdf-viewer` since it worked with version 4.1.2 and check which version broke it.

**Comment by stephanrauh** [comment]
The bugfix has landed with version 11.1.2.
